src/QMCDrivers/DMC/DMCOMP.cpp

```diff
@@ -327,7 +332,21 @@ bool DMCOMP::run()
         *(RandomNumberControl::Children[ip])=*(Rng[ip]);
     }
     recordBlock(block);
-  } while(block<nBlocks && myclock.elapsed()<MaxCPUSecs);
```

Comment:
Is myclock still used by others?

Reply:
myclock is not used elsewhere - it's local to this file.  I should remove it.